themes/SuiteP/include/MySugar/tpls/MySugar.tpl

```diff
@@ -98,7 +98,7 @@
 
         {if !$lock_homepage}
             <li id="tab-actions" class="dropdown">
-                <a class="dropdown-toggle" data-toggle="dropdown" href="#">{$dashboardActions}</a>
+                <a class="dropdown-toggle" data-toggle="dropdown" href="#">{$dashboardActions}{$APP.LBL_LINK_ACTIONS}</a>
                 {include file='themes/SuiteP/include/MySugar/tpls/actions_menu.tpl'}
```

Comment:
Should {$dashboardActions} maybe be removed/replaced by {$APP.LBL_LINK_ACTIONS}?

Reply:
@PedroErnst after that change it worked fine!